include/gsl/multi_span

```diff
@@ -44,6 +44,12 @@
 #pragma warning(disable : 4127) // conditional expression is constant
 #pragma warning(disable : 4702) // unreachable code
 
+// Turn off rules that generate too much noise. To be fixed in the static analysis tool.
```

Comment:
If you have some MS internal insights and expect these fixes to be in version x of MSVC then maybe we could only use them `#if _MSC_VER < x`.

Reply:
Not yet, will do when we have the fixes planned.